recipes/esys-escript/LICENSE-2.0.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This file is redundant as conda-build will use the one from the source tarball

Reply:
I've deleted the file.